### PR TITLE
add line 3 to julios.text

### PR DESCRIPTION
**Pull request by pauljct06**
i added a new line

### CODE REVIEW

File: thisIsJulio.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 this is julio
-julio is now touching this line. 
\ No newline at end of file
+julio is now touching this line.
+paul is touching line 3
```
Comment (julioc104) [inline thisIsJulio.txt:3]: why did you add this?

Comment (pauljct06) [reply thisIsJulio.txt:3]: this was a needed change

**Review by julioc104 — APPROVED** [review]
